pybase/region/client.py

```diff
@@ -291,4 +291,4 @@ def _send_hello(sock):
 def _to_varint(val):
     temp = []
     encoder(temp.append, val)
-    return b"".join(temp)
+    return "".join(temp).encode('utf8')
```

Comment:
you shouldn't need this with the new encoder... temp should be an array of bytes, so `b"".join(temp)` is ok